lib/ntpUtil.js

```diff
@@ -88,7 +88,7 @@ const prepareAssets = (jsonFileUrl, targetResourceDir, targetJsonFileName) => {
       }
       console.log(`Done - json file ${jsonFileUrl} parsing`)
       // Make sure the data has a schema version so that clients can opt to parse or not
-      const incomingSchemaVersion = photoData.schemaVersion
+      let incomingSchemaVersion = photoData.schemaVersion
```

Comment:
maybe we can change this around a bit and address @petemill comment- let me add a commit here 😄

Reply:
I went with a fallback to version 1 approach since we know what version 1 is now and we can try to load that